basic/nucleo/temperature/README.md

```diff
@@ -0,0 +1,166 @@
+
+# Temperature readings using NUCLEO-L152RE and Grove – Temperature&Humidity Sensor Pro
+
+This example measures temperature each minute. The temperature reading is shipped off to
+the things network over LoRaWAN.
+
+## Hardware
+
+![nucle](nucleo.jpg)
+
+- [NUCLEO-L152RE](https://developer.mbed.org/platforms/ST-Nucleo-L152RE/)
+- [Seeed Grove Shield V2 ](https://developer.mbed.org/components/Seeed-Grove-Shield-V2/)
+- [Grove – Temperature&Humidity Sensor Pro](http://www.seeedstudio.com/depot/grove-temperaturehumidity-sensor-pro-p-838.html)
+
+
+## Console output
+
+    Temperature is 23.00
+    Temperature is 23.00
+    Temperature is 23.00
+    Temperature is 23.00
+    Temperature is 23.00
+    Temperature is 23.00
+    Temperature is 23.00
+    Temperature is 23.00
+
+# Code
```

Comment:
Could you add the source code (copy or submodule) to the folder, so that people can just run it from this repository?